vadin-demo-gui/src/main/java/de/muenchen/vaadin/ui/components/forms/BuergerRWForm.java

```diff
@@ -0,0 +1,145 @@
+package de.muenchen.vaadin.ui.components.forms;
+
+import com.vaadin.ui.Component;
+import com.vaadin.ui.Field;
+import com.vaadin.ui.HorizontalLayout;
+import de.muenchen.vaadin.demo.i18nservice.buttons.SimpleAction;
+import de.muenchen.vaadin.guilib.components.BaseComponent;
+import de.muenchen.vaadin.guilib.components.buttons.ActionButton;
+import de.muenchen.vaadin.guilib.controller.EntityController;
+import de.muenchen.vaadin.ui.components.buttons.node.listener.BuergerSingleActions;
+import de.muenchen.vaadin.ui.components.forms.selected.SelectedBuergerForm;
+
+/**
+ * Provides a full-featured Form for a Buerger. It allows to view the current selected buerger and edit it.
+ *
+ * @author p.mueller
+ * @version 1.0
+ */
+public class BuergerRWForm extends BaseComponent {
+
+    /** The default (readOnly/write) mode the Form initially starts. */
+    public static final boolean DEFAULT_MODE = false;
```

Comment:
Was ist default? Write oder ReadOnly? In einer anderen Klasse hast du es auch READ_ONLY genannt, dass würde ich dann hier auch machen.